docs/content/contributing.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Contributing
+---
+
+Bug reports and pull requests are welcome on GitHub at [https://github.com/primer/view_components](https://github.com/primer/view_components). This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+
+## Setting up
+
+Run
+
+```bash
+script/setup
+```
+
+To install all necessary dependencies.
+
+## Running tests
+
+To run the full test suite:
+
+```bash
+bundle exec rake
+```
+
+## Writing documentation
+
+Documentation is written as [YARD](https://yardoc.org/) comments directly in the source code, compiled into Markdown via `rake docs:build` and served by [Doctocat](https://github.com/primer/doctocat).
+
+## Storybook / Documentation
+
+To run Storybook and the documentation site, run:
+
+```bash
+script/dev
+```
+
+_Note: Overmind is required to run script/dev._
```

Comment:
FYI we have a `Note` component in Doctocat that you can use if you want: https://primer.style/doctocat/components/note